src/state/machines/turn.ts

```diff
@@ -570,14 +572,23 @@ export const TurnMachine = setup({
             animationDuration: "#turn.checkingEventConditions.preDraw",
           },
         },
-        drawing: {
+        discardRow: {
```

Comment:
I think we should stick to the xstate standard for the naming of states. According to xstate these should be nouns or adjectives, not verbs, otherwise it is too easy to confuse them with actions

![image](https://github.com/user-attachments/assets/3ce106a5-bb67-4b48-ae99-6018fb201413)

Reply:
I'll change that